_policy/referees.md

```diff
@@ -1,15 +1,448 @@
+    globally as per the FIT Strategic Plan and Priorities..
+5.  Implement strategies for the implementation and ongoing development of referee personnel,
+    globally as per the FIT Strategic Plan and Priorities.
+6.  Must possess knowledge of the standards and behaviour expected of a FIT official and or
+    representative, and an acceptance of these standards as set out in the FIT Code of Conduct i.e. a
+    reference point for the standards.
+7.  Must possess a minimum of an Advanced Elite Certificate such as:
+8.  TFA Level 4 Referees Coaching Certificate;
+9.  TNZ Level 4 Referees Coaching Certificate; or 
+10. An accepted recognized equivalent within the Member NTA.
+11. Possess the ability to communicate effectively with a national or international body of referees
+    to update on current issues, trends and to disseminate information.
+12. Constantly revise the delivery of Referee Badge Level Courses and Referee Coaching Courses for
+    distribution globally.
+13. Regularly update all Member NTA's with any advancement in referee coaching techniques and/or
+    trends.
+14. Coordinate FIT Referee Commission Member visits and expected outcomes to Member NTA's.
+15. Implement a succession plan to develop referees and referee coaches, globally.
+16. Is a volunteer and not in the employment of FIT or any Member NTA.
+17. Must be non active as a referee (unless the appointment is to an Associate position for
+    developmental purposes.
+18. An outline role description for the FIT Director of Referees is attached as Appendix 1.
+
+### FIT Referee Commission Member Appointment & Criteria
+
+1.  The FIT Director of Referees and FIT Referees Commission Members are expected to be volunteers
+    of the sport and should not hold a full or part time position with any Member NTA. However, if a
+    Member NTA has a full or part time employee in the refereeing arm, that respective NTA can apply
+    to the FIT Secretary General to have the employee attend any FIT sanctioned event to use for
+    referee assessment, coaching  and developmental purposes.
+2.  It is envisaged that the Commission structure will contain a minimum of one Australian and one
+    New Zealand Member for the next five to ten years. It is further planned to have Commission
+    Members strategically placed through Asia, Europe and United Kingdom.
+3.  The role description for the FIT Referee Commission Member is attached as Appendix 2.
+4.  The FIT Referee Director will determine in consultation with the Secretary-General of FIT, the
+    FIT Referee Commission Member composition for Federation events. This takes into account that
+    team entries for a particular event and the costs involved in attending will need to be
+    considered in making a determination. An appointment to the FIT Referees Commission does not
+    automatically mean that s/he will attend all Federation events as FIT Referees Commission
+    Members.
+
+### Associate FIT Referee Commission Member
```

Comment:
We don't have a position description for an Associate Referee Commission Member in the list of position descriptions.

Is this even necessary? An association would just be another commission member, no?